IS0501Test.py

```diff
@@ -254,6 +254,20 @@ def test_09(self, test):
         else:
             return test.UNCLEAR("Not tested. No resources found.")
 
+    def test_09_01(self, test):
+        """All params listed in /single/senders/{senderId}/active/ match their corresponding SDP files"""
+
+        if len(self.senders) > 0:
+            for sender in self.senders:
+                if self.transport_types[sender] == "urn:x-nmos:transport:rtp":
+                    valid, response = self.is05_utils.check_sdp_matches_params(sender)

```

Comment:
Bit confusing calling it `response` because it's actually a string message, right?

Same applies a few times below.